doc/man3/RSA_generate_key.pod

```diff
@@ -19,13 +21,17 @@ Deprecated:
 
 =head1 DESCRIPTION
 
-RSA_generate_key_ex() generates a key pair and stores it in the B<RSA>
-structure provided in B<rsa>. The pseudo-random number generator must
-be seeded prior to calling RSA_generate_key_ex().
+RSA_generate_multi_prime_key() generates a key pair and stores it in the
+B<RSA> structure provided in B<rsa>. The pseudo-random number generator must
+be seeded prior to calling RSA_generate_multi_prime_key().
 
-The modulus size will be of length B<bits>, and the public exponent will be
-B<e>. Key sizes with B<num> E<lt> 1024 should be considered insecure.
-The exponent is an odd number, typically 3, 17 or 65537.
+RSA_generate_key_ex() is a wrapper of RSA_generate_multi_prime_key() with
+the B<primes> parameter set to 2.
+
+The modulus size will be of length B<bits>, the number of primes to form the
+modulus will be B<primes> and the public exponent will be B<e>. Key sizes
```

Comment:
comma after `B<primes>`.

Reply:
Addressed though I don't feel it's necessary...